SU-SWS/sulgryphon-nextjs
# SUL23-38 | @jdwjdwjdw | Build "Event" content display

# READY FOR REVIEW

# Summary
- [SUL23-38](https://stanfordits.atlassian.net/browse/SUL23-38): FE-DEV | Build "Event" content display in NextJS

# Review By (Date)

# Urgency

# Steps to Test
1. Add a few example events, testing out all the fields and time options
- all day event
- same day but different hours
- event that spans multiple days
2. Add a basic page with Events lists (default, card, past)
3. Confirm that the Events node page, lists, and cards all look correct.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [SUL23-38](https://stanfordits.atlassian.net/browse/SUL23-38): FE-DEV | Build "Event" content display in NextJS
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)


<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/31"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
